internal/setup/setup_test.go

```diff
@@ -822,7 +799,7 @@ var setupTests = []setupTest{{
 		},
 	},
 }, {
-	summary: "Archives with keyrings",
+	summary: "Archives with public keys",
```

Comment:
We need a test with no public keys and we need to raise an error if that happens. We expect the archive to provide at least a public key in its YAML definition.

Reply:
The tests will all need to contain public keys except the one mentioned above.

Reply:
Okay, seems like a good idea.